src/sync/installation.ts

```diff
@@ -225,6 +432,12 @@ export const processInstallation =
 				jiraHost
 			});
 
+			if (await booleanFlag(BooleanFlags.SPLIT_PROCESS_INSTALLATION_FUNCTION, false, jiraHost)) {
```

Comment:
I noticed this line twice: `await booleanFlag(BooleanFlags.SPLIT_PROCESS_INSTALLATION_FUNCTION, false, jiraHost`. After line 416 can we do something like:

```
const splitProcessInstallationFunctionFlagIsOn = await booleanFlag(
				BooleanFlags.SPLIT_PROCESS_INSTALLATION_FUNCTION,
				true,
				jiraHost
			);
```

and then pass that variable?

- [ ] move call to FF to variable